app/src/main/java/com/cooper/wheellog/MainActivity.kt

```diff
@@ -549,8 +549,13 @@ class MainActivity : AppCompatActivity() {
                     miSearch!!.isEnabled = true
                     miSearch!!.icon!!.alpha = 255
                 }
-                miLogging!!.isEnabled = false
-                miLogging!!.icon!!.alpha = 64
+                if (LoggingService.isInstanceCreated()) {
+                    miLogging!!.isEnabled = true
```

Comment:
I was thinking about this since I got the notification for the merge. I'm not sure this branch is needed. I think it's already gonna be enabled and have its alpha set to 255, so all we need to do is change the settings if `LoggingService.isInstanceCreated()` is false.